pkg/nodeconfig/nodeconfig.go

```diff
@@ -39,6 +39,11 @@ import (
 	"github.com/openshift/windows-machine-config-operator/version"
 )
 
+const (
+	// WMCONamespace defines the namespace WMCO-managed resourced are expected to be in
+	WMCONamespace = "openshift-windows-machine-config-operator"
```

Comment:
I don't see how this belongs in nodeconfig.
In any case, you should use the watchNamespace value thats present in each controller instead of using a constant.